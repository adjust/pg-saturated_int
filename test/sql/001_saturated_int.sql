create extension saturated_int;

select 999999999999999::saturated_int;
select 2147483648::saturated_int;
select 2147483647::saturated_int;
select (-2147483649)::saturated_int;
select (-2147483648)::saturated_int;

