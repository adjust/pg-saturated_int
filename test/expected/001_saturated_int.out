create extension saturated_int;
select 999999999999999::saturated_int;
 saturated_int 
---------------
 2147483647
(1 row)

select 2147483648::saturated_int;
 saturated_int 
---------------
 2147483647
(1 row)

select 2147483647::saturated_int;
 saturated_int 
---------------
 2147483647
(1 row)

select (-2147483649)::saturated_int;
 saturated_int 
---------------
 -2147483648
(1 row)

select (-2147483648)::saturated_int;
 saturated_int 
---------------
 -2147483648
(1 row)

